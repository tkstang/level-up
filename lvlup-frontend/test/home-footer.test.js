import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from '../src/components/Home/footer';

test('Footer should render', () => {
  const home = shallow(
    <Footer />,
  );
  expect(toJson(home)).toMatchSnapshot();
});
