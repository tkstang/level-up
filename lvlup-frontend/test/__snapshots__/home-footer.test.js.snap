exports[`test Footer should render 1`] = `
<div
  className="footer">
  Contact Us     ||   Enter   ||    To the Top   
</div>
`;
